tools/lkl/include/lkl.h

```diff
@@ -461,6 +461,21 @@ int lkl_if_wait_ipv6_dad(int ifindex, void *addr);
  */
 int lkl_set_fd_limit(unsigned int fd_limit);
 
+/**
+ * lkl_sysctl - write a sysctl value
+ *
+ * @path - the path to an sysctl entry (e.g., "net.ipv4.tcp_wmem");
+ * @value - the value of the sysctl (e.g., "4096 87380 2147483647")
+ */
+int lkl_sysctl(char *path, char *value);
+
+/**
+ * lkl_sysctl_parse_write - Configure sysctl parameters with strings
+ *
+ * @sysctls - Configure sysctl parameters as the form of "key|value;..."
+ */
+void lkl_sysctl_parse_write(char* sysctls);
```

Comment:
should this function be just in init.c? I think only hijack needs it and it's confusing to have it in lkl.h